packages/core/react/src/WalletProvider.tsx

```diff
@@ -49,7 +49,7 @@ export function WalletProvider({
     onError,
 }: WalletProviderProps) {
     const { connection } = useConnection();
-    const adaptersWithStandardAdapters = useStandardWalletAdapters(adapters);
+    const adaptersWithStandardAdapters = useStandardWalletAdapters(adapters as Adapter[]);
```

Comment:
would require same improvement for useStandardWalletAdapters of @solana/wallet-standard-wallet-adapter-react